source/develop/release-management/features/network/networkfilterparameters.html.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Network Filter Parameters
+category: feature
+authors: dholler
+wiki_category: Feature
+wiki_title: Network/NetworkFilterParameters
+wiki_revision_count: 0
+wiki_last_updated: 2016-11-08
+feature_name: Network Filter Parameters
+feature_modules: engine,vdsm
+feature_status: In Development
+---
+
+# Network Filter Parameters
+
+## Summary
+
+OVirt already supports filtering network communication of VMs.
+The feature introduced here adds the ability to configure this filters by parameters.
+
+## Owner
+
+*   Name: [Dominik Holler](User:dominikholler)
+
+*   Email: <dholler@redhat.com>
+
+## Detailed Description
+
+The VM network communication can by restricted using the already available feature called [network filters][2].
+The feature introduced here adds the ability to configure these filters by [parameters][3].
+oVirt allows to manage the network filter parameters and VDSM uses the libVirt to apply them to the VMs configuration.
+
+In the scenario of [Bug 1366905][4] network filter parameters are required to restrict the allowed IP addresses of a VM.
+Network filter parameters can not only be used to define allowed or blocked IP address,
+but also for restricting port ranges on transport layer or allowing only trusted DHCP servers.
+
+## Benefit to oVirt
+
+This feature enables the usages of oVirt in scenarios, in which the user has to set parameters,
+without forcing the user to edit configuration files on the hosts manually.
+
+### User work-flows
+
+To use network filter parameters, the following workflow is expected:
+
+1. The user may create a new logical network or decide to use an already existing one.
+2. The user may create a new vNIC Profile or use an existing one.
+3. The user ensures, that the logical network is assigned to the hosts of the cluster.
+4. The user instantiates a (virtual) network interface (vNIC) of a virtual machine by picking the vNIC profile. 
+5. The user configures applied vNIC Profile to use a network filter, which depends on parameters.
+6. The user creates network filter parameters for this network interface of the virtual machine.
+
+If the predefined logical network `ovirtmgmt` and the associated predefined vNIC Profile `ovirtmgmt` is used,
+there is nothing to do in the first three steps.
+
+### User Experience
+
```

Comment:
Hi Dominik

I think we should add a mockup/screenshot of the associate vNIC profile, which you called 'clean traffic' in your example. In the sceenshot we can show that you choosing 'clean traffic' Network filter for the vNIC profile.
Without it, it's a bit hard to understand the flow, if we only showing the VM network interface dialog.